### DISCUSSION TITLE
mixing low res and high res

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#13609] [Ideas]
First things first, thank you to everyone involved in improving Frigate. Version 0.14 is amazing!

I am running version 0.14.1 on an Aoostar R1 mini pc (intel n100 with 16 GB ram) and I've notice that the main page seems to use a considerable amount of bandwidth. I've used the ```live -> stream_name``` functionality to force it to use the low res stream (https://docs.frigate.video/configuration/live/#setting-stream-for-live-ui). However, when I click on a camera's view to enlarge it, the view remains on the low res stream. Is it possible to make Frigate display the high res stream when viewing a single camera? In other words, when I click on a live view of a camera to expand it to fill the screen, Frigate continues to use the low res stream, but I would like it to switch over to use the high res stream.

Thank you!

**Comment by NickM-27** [comment]
it's not possible currently, this is something that will require a config change and would be a new feature we have to implement

**Reply by Cold-Lemonade** [reply]
I would love to see this new feature. If I knew how to program, I would volunteer for the task. But alas, I do not.

**Reply by NickM-27** [reply]
It's probably only a couple lines of code. The main thing is just how to make the config easiest and how to migrate existing config.
